sktime/transformers/series_as_features/paa_multivariate.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+from sktime.transformers.series_as_features.base import \
+    BaseSeriesAsFeaturesTransformer
+from sktime.utils.validation.series_as_features import check_X
+from sktime.transformers.series_as_features.dictionary_based._paa import PAA
+
+__author__ = "Vincent Nicholson"
+
+"""
+This class is essentially a wrapper for the _paa
+transformer but for multivariate data.
+Intended for use by ShapeDTW.
+"""
+
+
+class PAA_Multivariate(BaseSeriesAsFeaturesTransformer):
```

Comment:
is this fundamentally different from the univariate version? No way to combine the two into a single transformer?

Reply:
Yeah that's a good suggestion I'll modify the original one so that it supports multivariate. Therefore, it'll do PAA along each dimension :)